Bindings/Java/Matlab/Dynamic_Walking_Challenge/Dynamic_Walker_Simulation/walker_simulation_objective_function.m

```diff
@@ -0,0 +1,69 @@
+function J = walker_simulation_objective_function(vod)
+%% walker_simulation_objective_function.m
+% Function for running forward simulation of a walker model and computing 
+% the objective function (Pelvis X translation). The input to this function 
+% is a 12x1 vector of initial coordinate value and speed states for a 
+% dynamic walker model. See Matlab script walkerOptimization.m for use.  
+
+% Written by James Dunne, Carmchael Ong, Tom Uchida, Ajay Seth. 
+
+% Define Global variables. 
+global model translation_store vectorOfDefaults_store;
+
+% Import OpenSim libraries 
+import org.opensim.modeling.*
+
+% Store the vector of defaults that the optimizer chooses
+vectorOfDefaults_store = [vectorOfDefaults_store;vod']; 
+
+% Set the time for the simulation to take place  
+finalTime = 10;
+
+% Update model default coordinate values and speeds with optimizer defined 
+% values.
+model.getCoordinateSet().get('Pelvis_tx').setDefaultValue(vod(1));
+model.getCoordinateSet().get('Pelvis_tx').setDefaultSpeedValue(vod(2));
```

Comment:
I don't think the speed of `Pelvis_tx` should be a design variable in the optimization problem: the optimizer should be able to just launch the model forward.

Reply:
Also, perhaps initial conditions for `Pelvis_tx`, `Pelvis_ty`, and `Pelvis_ty` speed should not be optimized?